sys/include/net/gnrc/netif/lorawan.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief  LoRaWAN adaption for @ref net_gnrc_netif
+ *
+ * @author  Jose Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ */
+#ifndef NET_GNRC_NETIF_LORAWAN_H
+#define NET_GNRC_NETIF_LORAWAN_H
+
+#include "net/gnrc/lorawan.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   A Link Check request was scheduled
+ */
+#define GNRC_NETIF_LORAWAN_FLAGS_LINK_CHECK                (0x00020000U)
```

Comment:
Still doesn't fit in `gnrc_netif_lorawan_t::flags` ;-). It doesn't have any other flags yet, how about `0x01`?

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/11022/commits/28f641154386b6a021365ca0c4ddb534ff8e1308